src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator {
+    private String color;
+
+    public String getColor() {
+        return color;
+    }
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
+    @Override
+    public double getArea() {
+        return 0;
+    }
+
+    public void draw() {

```

Comment:
Please create an interface that will provide this operation.

Reply:
ok, done